### ISSUE TITLE
negative added_reads

### ISSUE THREAD

**Issue by choon-sim** [jenniferlu717/Bracken#141]
Hi, I am curious to know whether it is possible the 'added_reads' in Bracken output can be a negative number.

name	taxonomy_id	taxonomy_lvl	kraken_assigned_reads	added_reads	new_est_reads	fraction_total_reads
Phocaeicola vulgatus	821	S	82772	-19082	63690	0.03416

Thanks,

**Comment by jenniferlu717** [comment]
It is possible in a strange situation with some adjustments I made to the script. I'll try to describe it here:

Assume you have species A with strains 1 and 2 and related species B with strain 3, 

but for some reason, reads from strains 1 and 2 classify as species B (meaning they might be more related to strain 3 and probably should be in the same species with strain 3), then when Bracken reorganizes reads, it will take away reads from species A and give them to species B. 

This is a bit of a strange situation that does happen, but not all the time.

**Comment by meenachakra** [comment]
Hi! I and a collaborator are a bit confused about this - we thought that Bracken just moves reads from higher levels down, and reads from lower levels up. We didn't think there was any horizontal movement of reads?

**Comment by salzberg** [comment]
@meenachakra you are correct, Bracken just moves reads down or up in the hierarchy, it does not move them sideways. Jen is referring to an unusual case where the actual data might include a species that has some sequence matching a different species (or genus). *However* we just discovered a bug, which Jen is fixing right now (already partly fixed), that causes Bracken to "zero out" some species when there are multiple near-identical strains belonging to that species. The updated files will be posted soon (and two updates are already posted).

**Comment by salzberg** [comment]
Update: the bug is fixed and Bracken no longer has the problem where it can "zero out" species in a custom taxonomy. Users should update their Bracken version to the latest github release.

**Comment by meenachakra** [comment]
Thank you so much!

**Comment by TheSallyGardens** [comment]
> salzberg
Hi！
How to get unique reads from bracken?
